website/docs/d/elastic_beanstalk_environment.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+subcategory: "Elastic Beanstalk"
+layout: "aws"
+page_title: "AWS: aws_elastic_beanstalk_environment"
+description: |-
+  Retrieve information about an Elastic Beanstalk Environment
+---
+
+# Data Source: aws_elastic_beanstalk_environment
+
+Retrieve information about an Elastic Beanstalk Environment.
+
+## Example Usage
+
+```terraform
+data "aws_elastic_beanstalk_environment" "example" {
+  name = "my-app"
```

Comment:
I know that it is only an example, but I'd rather use `my-environment` instead of `my-app` as `name` attribute refers to environment's name and `application` attribute refers to app name.